Reading List/Reading List/View Controllers/ReadingListTableViewController.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  ReadingListTableViewController.swift
+//  Reading List
+//
+//  Created by Norlan Tibanear on 6/6/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class ReadingListTableViewController: UITableViewController, BookTableViewCellDelegate {
+    
+    
+    // Variables
+    var bookController = BookController()
+    
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.reloadData()
+    }
+    
+    
+    override func viewDidAppear(_ animated: Bool) {
+        tableView.reloadData()
```

Comment:
I would likely remove the reload call from viewDidLoad, as viewDidAppear is called shortly after making the first call somewhat redundant. Not sure if it makes any difference in this application, but you may want to use viewWillAppear to reload data as it gets called earlier than viewDidAppear